### PR TITLE
Rename some of the increment actions to noop.

### PR DESCRIPTION
**Pull request by nixpulvis**
I'm just starting to look at this library, but reading the README this stood out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,11 +48,11 @@ redux-actions will automatically set ```action.error``` to true.
 Example:
 
 ```js
-const increment = createAction('INCREMENT');
+const NOOP = createAction('NOOP');
```
Comment (yangmillstheory) [inline README.md]: Can we follow the previous casing?

**Comment by yangmillstheory** [comment]
Thanks. This was likely copy-pasta.

File: README.md (hunk 1 above)
Comment (nixpulvis) [inline README.md]: LOL I suck at programming.

File: README.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ Wraps an action creator so that its return value is the payload of a Flux Standa
 Example:
 
 ```js
-let increment = createAction('INCREMENT', amount => amount);
+let noop = createAction('NOOP', amount => amount);
```
Comment (devdigital) [inline README.md]: Why is noop more suitable than increment for the action creator name? as the action created from this action creator is used to increment a counter?

File: README.md [hunk 3]
```diff
@@ -63,17 +63,17 @@ expect(increment(error)).to.deep.equal({
...
-handleAction(increment, {
+handleAction(noop, {
   next(state, action) {...},
   throw(state, action) {...}
 });
 
 // As object key in handleActions:
 const reducer = handleActions({
-  [increment]: (state, action) => ({
+  [noop]: (state, action) => ({
```
Comment (devdigital) [inline README.md]: Should this not be the action creator name (i.e. 'INCREMENT' or a const) rather than the action creator function?

**Pull request merged**